exercises/hello-world/.meta/.version

```diff
@@ -1 +1 @@
-1
\ No newline at end of file
+2
```

Comment:
Clarification on why I believe it needs to be incremented is that old version would take an argument, this new version will not accept an argument.